src/config.erl

```diff
@@ -183,6 +187,26 @@ delete(Section, Key, Persist, Reason) when is_list(Section), is_list(Key) ->
     gen_server:call(?MODULE, {delete, Section, Key, Persist, Reason}).
 
 
+features() ->
+    lists:usort([list_to_binary(Key) || {Key, "true"} <- ?MODULE:get(?FEATURES)]).
+
+
+feature_enable(Feature) when is_atom(Feature) ->
```

Comment:
`enable_feature` and `disable_feature` reads better, what do you think? Also, can we insist on atom rather than handle atom and binary and list?